src/aof.c

```diff
@@ -768,6 +768,7 @@ int openNewIncrAofForAppend(void) {
     if (ret == C_ERR) {
         close(newfd);
         aofManifestFree(temp_am);
+        sdsfree(new_aof_name);
```

Comment:
same as above

Reply:
Thanks, I just notice it. I will close this PR, great work.